test/extensions/filters/http/common/fuzz/filter_fuzz_test.dict

```diff
@@ -0,0 +1,2 @@
+"\x0A\x04\x08\x80\x80\x40" 
```

Comment:
Can you provide documentation or developer guidance for how to generate these and grow this corpus?

Reply:
Can I mark this as a TODO once the dictionary PR is passed? Since every http filter has a unit test (`buffer_filter_test.cc`, `header_to_metadata_filter.cc`, `health_check_test.cc`) I think an easy way to tackle this would be to:
- Configure a base FilterTest class that has member variable `config_`, `file_name_` (all tests already have a filter config shared ptr) with a custom TearDown test that executes the code I  used to generate the strings:
```
    // Create custom path with test environment variable
    envoy::extensions::filters::network::http_connection_manager::v3alpha::HttpFilter filter;
    filter.set_name(filter_name);
    filter.mutable_typed_config()->PackFrom(proto_config);
    google::protobuf::TextFormat::Printer printer;
    printer.SetExpandAny(false);
    printer.SetUseUtf8StringEscaping(false);
    std::string output;
    printer.PrintToString(filter, &output);
    ENVOY_LOG_MISC(info, "writing {} to {}", output, path);
```
- Move all existing filter tests to extend the FilterTest

Reply:
Yeah, TODO and tracking issue is fine, thanks.